templates/travel/partials/icon/boat.snip.html

```diff
@@ -0,0 +1,21 @@
+{{!
+<!---
+Copyright 2016 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+<!-- Boat Icon -->
+<svg class="travel-icon" viewBox="0 0 100 100"><path fill="currentColor" d="M91.573 93.997c-2.926.06-5.816-.704-8.377-2.22-3.313-1.927-7.315-1.927-10.63 0-5.24 2.96-11.51 2.96-16.752 0-1.61-1.006-3.45-1.507-5.314-1.446-1.864-.06-3.706.44-5.315 1.448-2.56 1.515-5.45 2.28-8.376 2.22-2.927.06-5.817-.705-8.378-2.22-1.61-1.007-3.45-1.508-5.314-1.447-1.864-.06-3.706.44-5.315 1.448-2.56 1.515-5.45 2.28-8.376 2.22C7.535 93.997 6 92.355 6 90.33s1.535-3.665 3.427-3.665c1.864.062 3.705-.44 5.314-1.445 2.562-1.515 5.452-2.28 8.378-2.22 2.926-.06 5.815.705 8.376 2.22 1.61 1.006 3.45 1.508 5.315 1.445 1.862.062 3.705-.44 5.314-1.445 2.56-1.515 5.45-2.28 8.376-2.22 2.926-.06 5.815.705 8.376 2.22 3.314 1.927 7.316 1.927 10.63 0 5.24-2.958 11.51-2.958 16.75 0 1.61 1.006 3.453 1.507 5.317 1.445 1.892 0 3.427 1.64 3.427 3.665 0 2.024-1.535 3.665-3.427 3.665zM90.293 62h-30.89c-.373 0-.73-.15-.993-.413-.263-.264-.41-.623-.41-.996 0-.19.037-.377.112-.552 2.28-5.897 3.39-12.185 3.262-18.51.12-6.36-.988-12.683-3.264-18.62-.073-.17-.11-.352-.11-.537 0-.38.16-.743.44-1.003.277-.26.65-.39 1.03-.364 17.262 1.17 31.527 22.68 32.527 39.178.028.473-.14.936-.46 1.28-.324.344-.774.54-1.245.54zm-1.746 4c.973.002 1.853.578 2.245 1.47.392.89.22 1.93-.436 2.648-2.933 3.114-6.198 5.897-9.736 8.3-1.1.75-2.394 1.162-3.724 1.185-3.243.037-6.427.864-9.278 2.41-2.33 1.31-5.173 1.308-7.502-.004-3.01-1.636-6.388-2.467-9.81-2.414-3.427-.052-6.808.78-9.82 2.417-1.122.695-2.427 1.037-3.746.98-1.322.056-2.63-.287-3.754-.984-3.01-1.636-6.388-2.467-9.812-2.415-2.76-.05-5.5.49-8.036 1.578-.47.207-1.014.163-1.446-.118-.43-.28-.69-.76-.692-1.275V68.55c0-.677.267-1.325.745-1.803.477-.478 1.124-.747 1.8-.747h73.002zm-37.12-4H7.715c-.67 0-1.28-.392-1.56-1.004-.278-.612-.175-1.332.265-1.84L50.99 7.593c.47-.542 1.224-.735 1.894-.485.67.25 1.115.893 1.116 1.61v50.703c0 .685-.272 1.34-.754 1.825-.482.484-1.136.755-1.818.754z"/></svg>
```

Comment:
svgs should go inside images directory, they cant be here and they cant be .html. 

But , you can include them as partials (if you had done this in your previous PR please fix those as well.)

Reply:
and use `svgo` to optimize all your svgs.